R/deploySite.R

```diff
@@ -77,17 +80,17 @@ deploySite <- function(siteDir = getwd(),
     appFiles <- appFiles[!grepl("^.*\\.([Rr]|[Rr]md|md)$", appFiles)]
   }
 
-  # determine appSourceDoc
-  if (file.exists("index.Rmd"))
-    appSourceDoc <- "index.Rmd"
-  else
-    appSourceDoc <- "index.md"
+  if (is.null(recordDir)) {
+    # recordDir should usually be an absolute path, but above we set the
```

Comment:
Perhaps `deployApp` should `normalizePath(recordDir)`, as it does for `appDir`.

Reply:
Oh I can just use `file.path()` to make it explicit. This doesn't imply a change of behaviour because we were previously passing a path relative to the working directory, which is set to `siteDir`.